examples/run_nutation.py

```diff
@@ -82,16 +86,18 @@
         SW_kHz=config_dict["SW_kHz"],
         ret_data=data,
     )
-data.setaxis("indirect", p90_range_us * 1e-6).set_units("indirect", "s")
+data.rename("indirect", "beta")
+data.setaxis("beta", beta_range_s_sqrtW).set_units("beta", "s√W")
+data.set_prop("p_90s", prog_p90_us)
 # {{{ chunk and save data
-data.chunk("t", ["ph1", "t2"], [4, -1])
-data.setaxis("ph1", ph1_cyc / 4)
+data.chunk("t", ["ph2", "ph1", "t2"], [2, 2, -1])
```

Comment:
Are you sure that this ordering is correct?

Reply:
and with testing/processing it shows up in the right pathway